Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -2269,9 +2277,22 @@ extension BrowserViewController {
               presentedViewController == nil,
               !showLoadingScreen(for: .shared) else { return }
         
-        if !presentDefaultBrowserPromoIfNeeded() {
-            presentWhatsNewPageIfNeeded()
-        }
+        // TODO: To review this logic as part of the upgrade
+        /*
+         We are not fan of this one, but given the current approach a refactor
```

Comment:
Agreed and understood 😁

Reply:
I'm also keen to have it removed at some point soon